### ISSUE TITLE
Caption buttons are inverted when using dir="rtl"

### ISSUE THREAD

**Issue by ebrahim-2** [gpbl/react-day-picker#1739]
### Bug description

when using the `numberOfMonths` prop to render more than one calendar and using `dir='rtl'` prop the caption navigation buttons don't change position, go to the previous and next month buttons changes the icon direction but does not change position 

#### Steps

1. render more than one calendar
2. add rtl direction

### Expected behavior

go to previous month button should be on the right and go to next month should be on the left

### Screenshots

![image](https://user-images.githubusercontent.com/32773511/228953759-097c5820-e9cb-4cb8-9a0c-764b5e04e481.png)
![image](https://user-images.githubusercontent.com/32773511/228953845-b43995e1-e675-4d55-b029-a63fa62fb276.png)

### Additional context

when not using `numberOfMonths` prop the navigation buttons work as expected and are in the right position
